### PR TITLE
Migrate to MPS 2022.2

### PR DESCRIPTION
**Pull request by HeikoBecker**
This PR runs the necessary migrations to update mbeddr.formal to MPS 2022.2

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -25,11 +25,21 @@ jobs:
...
-        java-version: 11  
-        
+        java-version: 17  
+      
+    - name: Resolve MPS
+      uses: gradle/gradle-build-action@v2.4.2
+      with:
+        arguments: resolveMps -Pgpr.user=${{github.actor}} -Pgpr.token=${{ secrets.GITHUB_TOKEN }}
+
+    - name: Report MPS Integrity
```
Comment (sergej-koscejev) [inline .github/workflows/main.yml:38]: I think we should clean up experimental/debugging stuff like this.

Comment (HeikoBecker) [reply .github/workflows/main.yml]: I removed the integrity check. But I could not find any other changes similar to that. I guess this is ready for merge then if the CI succeeds.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -25,11 +25,16 @@ jobs:
...
        
-    - name: Set up JDK 11
+    - name: Set up JDK 17
       uses: actions/setup-java@v1
       with:
-        java-version: 11  
-        
+        java-version: 17  
+      
+    - name: Resolve MPS
```
Comment (sergej-koscejev) [inline .github/workflows/main.yml:33]: I think we also don't need this step, it should be part of the later build.

Comment (HeikoBecker) [reply .github/workflows/main.yml:33]: I have fixed it in a separate branch `featurer/migration_leftovers`.

**Pull request merged**